### DISCUSSION TITLE
how to enable two factor auth  in github

### DISCUSSION THREAD

**Discussion by BLACKHOODIE-pro** [community/community#143503] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

answers ??

**Comment by Sikorsky3301** [comment]
from the setting panel

**Comment by ebndev** [comment]
Hi @BLACKHOODIE-pro & @Sikorsky3301, thanks for being a part of the GitHub Community!

While we appreciate you wanting to contribute to Discussions, please note: We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

As a result, we'll be unmarking the answer and locking this, and other related posts. 

Any future violations may result in a temporary or indefinite block from the Community. 

Thanks for understanding and we hope to see you participate in other discussions in the future!
